engines/python/setup/djl_python/transformer-neuronx.py

```diff
@@ -47,7 +47,10 @@ def convert_opt(self, amp):
         self.model = AutoModelForCausalLM.from_pretrained(
             self.model_id_or_path, low_cpu_mem_usage=True)
         logging.info(f"Start model conversion to INF2 format...")
-        load_path = os.path.join(tempfile.gettempdir(), self.model_id_or_path)
+        prefix = self.model_id_or_path
+        if os.path.exists(prefix):
+            prefix = "converted_model"
+        load_path = os.path.join(tempfile.gettempdir(), prefix)
```

Comment:
```suggestion
        load_path = os.environ.get("SERVING_DOWNLOAD_DIR")
        if not load_path:
            load_path = tempfile.gettempdir()

        load_path = os.path.join(load_path, prefix)
```

Reply:
Neuron API will try to make dir with given path. If folder exist, it will fail

Reply:
so we have to make the directory unique